mindmeld/components/question_answerer.py

```diff
@@ -256,23 +304,67 @@ def load_kb(
                 connection to the Elasticsearch host.
             clean (bool): Set to true if you want to delete an existing index
                 and reindex it
+            app_path (str): The path to the directory containing the app's data
+            config (dict): The QA config if passed directly rather than loaded from the app config
         """
+        embedder_model = None
+        embedding_fields = []
+        if not app_path and not config:
+            logger.warning(
```

Comment:
Nit: Shouldnt this warning be localized to when devs use the embedder or not when they use regular ES?

Reply:
Not sure I understand - the only way to tell if someone is trying to use the embedder is if they provide an application config. How would we filter cases for which to give this warning?

Reply:
From my understanding, this warning will occur in the most common scenario where the user is just using `{"model_type": "keyword"}` and does not provide `app_path `. In this case, the warning of `You must provide either the application path to upload embeddings as specified  in the app config or directly provide the QA config` doesn't concern him/her since they didnt intend to use an embedding.

Reply:
If they did not provide either an app_path or a config, there is no way to know that they have specified `{"model_type": "keyword"}`